docs/docs/connect/data-source/sqlite.md

```diff
@@ -11,11 +11,19 @@ sidebar_position: 80
 
 [SQLite](https://www.sqlite.org/about.html) is a lightweight, self-contained SQL database engine renowned for its reliability, speed, and full-featured, serverless architecture. SQLite is primarily known as an in-process database and is widely used in embedded systems, mobile applications, and various small-to-medium-sized applications due to its simplicity, zero-configuration, and single-file database format. SQLite supports standard SQL syntax and includes features such as transactions and atomic commit and rollback, making it a practical choice for applications requiring a compact, efficient data management system. Rill supports connecting to and reading from a SQLite database as a source through the [DuckDB SQLite extension](https://duckdb.org/docs/extensions/sqlite.html).
 
-<img src='/img/connect/data-sources/sqlite.png' class='rounded-gif' style={{width: '75%', display: 'block', margin: '0 auto'}}/>
-<br />
 
 
-## Local credentials
+
+## Connect to SQLite
+```yaml
+# Connector YAML
+# Reference documentation: https://docs.rilldata.com/reference/project-files/connectors
+
+type: connector
+driver: sqlite 
+
+dsn: "file:mydatabase.db" 
+```
```

Comment:
(Same principle as for images/GIFs) We should not start a section with a code block; a preamble should precede any code block.